string service_command_name
string service_command_value
---
bool service_response